### PR TITLE
Wagashi review1226

### PR DESCRIPTION
**Pull request by oseromio**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,40 +2,42 @@
...
-    <meta name="keywords" content="和菓子,動物,お菓子,EC">
-    <link rel ="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
-    <link rel="stylesheet" href="stylesheet.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <meta charset="utf-8">
+  <title>wagashizoo</title>
+  <meta name="description" content="動物の形をしたかわいい和菓子のECサイトです。">
+  <meta name="keywords" content="和菓子,動物,お菓子,EC">
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet">
```
Comment (michiminstar) [inline index.html]: CSSを読み込む順番は、normalize.cssが一番上がいい。次にbootstrap.min.css

File: index.html [hunk 2]
```diff
@@ -49,34 +51,52 @@
...
-        <div class="container">
-            <p class="text">わがしzoo</p>
-            <p class="detail">動物をかたどった和菓子を専門にしています。</p>
-            <p class="text">どうぶつをさがす</p>
-            <p class="detail">空の動物・陸の動物・河の動物・・・。<br>
-                              かわいい動物和菓子達があなたを待っています。<br>
-                              あなたが好きな動物はどこ住んでいますか？</p>
-       <!-- </div> -->
-     </div>
+      <div id="carousel-topimg wagashi-topimg">
```
Comment (michiminstar) [inline index.html:55]: 名前は「carousel-top-img」とかがいいな。ちなみにここid複数指定した理由ある？
なければ、片方はidで、もう片方はclassでもいいと思う。

File: index.html [hunk 3]
```diff
@@ -49,34 +51,52 @@
...
+      <div class="container">
+        <p class="text">わがしzoo</p>
+        <p class="detail">動物をかたどった和菓子を専門にしています。</p>
+        <p class="text">どうぶつをさがす</p>
+        <p class="detail">空の動物・陸の動物・河の動物・・・。<br>
+                          かわいい動物和菓子達があなたを待っています。<br>
+                          あなたが好きな動物はどこ住んでいますか？</p>
+     <!-- </div> -->
+      <!-- </div> -->
+    </div>
```
Comment (michiminstar) [inline index.html:88]: おそらくこの辺でdivがおかしくなってる

**Review by michiminstar — COMMENTED** [review]
みおちゃん、スクリーンショットもお願いします！

**Comment by michiminstar** [comment]
チェックするコードの量が長くなってきたから、一旦mergeしちゃいましょう。
